core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -400,7 +400,10 @@ private[spark] class Executor(
       // Report executor runtime and JVM gc time
       Option(task).foreach(t => {
         t.metrics.setExecutorRunTime(TimeUnit.NANOSECONDS.toMillis(
-          System.nanoTime() - taskStartTimeNs))
+          // SPARK-32898: it's possible that a task be killed before it reaches
+          // "taskStartTimeNs = System.nanoTime()". In this case, the executorRunTime
```

Comment:
`before it reaches "taskStartTimeNs = System.nanoTime()"` might be a little misleading because this looks like equality. If we want to pointing [line 473](https://github.com/apache/spark/pull/29789/files#diff-5a0de266c82b95adb47d9bca714e1f1bR473), can we use a different wording?

Reply:
Thanks, addressed.